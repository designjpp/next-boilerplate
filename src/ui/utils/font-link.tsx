/* eslint-disable @next/next/no-css-tags */
export const FontLink = () => (
	<>
		<link rel='preconnect' href='https://css.gstatic.com/' crossOrigin='true' />
		<link rel='preload' as='style' href='/css/font.css' />
		<link rel='stylesheet' href='/css/font.css' />
	</>
);
