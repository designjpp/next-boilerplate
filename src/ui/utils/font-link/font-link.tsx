import Head from 'next/head';

/* eslint-disable @next/next/no-css-tags */
export const FontLinkHead = () => (
	<Head>
		<link rel='preconnect' href='https://css.gstatic.com/' crossOrigin='true' />
		<link rel='preload' as='style' href='/css/font.css' />
		<link rel='stylesheet' href='/css/font.css' />
	</Head>
);
