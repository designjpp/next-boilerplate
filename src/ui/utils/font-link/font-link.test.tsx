import { create } from 'react-test-renderer';

import { FontLinkHead } from './font-link';

describe('<FontLinkHead>', () => {
	// Forgive me. The results of any content in Head is null,
	// and I couldn't find any good solution so this test is
	// redundant  and it's just for keeping coverage up
	it("should matches it's snapshots", () => {
		const fontLinks = create(<FontLinkHead />).toJSON();
		expect(fontLinks).toMatchSnapshot();
	});
});
