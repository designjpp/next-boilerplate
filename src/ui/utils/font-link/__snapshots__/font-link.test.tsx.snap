// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FontLinkHead> should matches it's snapshots 1`] = `null`;
